app/src/androidTest/java/net/aiscope/gdd_app/SampleDtoKtTest.kt

```diff
@@ -0,0 +1,28 @@
+package net.aiscope.gdd_app
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.aiscope.gdd_app.model.Sample
+import net.aiscope.gdd_app.network.toDto

```

Comment:
Maybe we could use an import alias here and tackle the ambiguity of the names?
`import net.aiscope.gdd_app.network.toDto as toDtoNetwork`

Reply:
Adds complexity no? I wasn't planning to test the other method here as it already has some coverage